frontend_vue/src/composables/useValidation.ts

```diff
@@ -0,0 +1,48 @@
+import * as Yup from 'yup';
+import { TOKENS_TYPE } from '@/components/constants.ts';
+import { ref } from 'vue';
+import type { Ref } from 'vue';
+
+type FieldsType = {
+  email: string;
+  memo: string;
+};
+
+type ValidateSchemaType = {
+  [key: string]: {
+    schema: Yup.ObjectSchema<FieldsType>;
+  };
+};
+
+export function useValidation() {
```

Comment:
again, any reason to export a function here?

Reply:
at the current stage, not really! 
I'll make them all utilities 🙏